mllib/src/main/scala/org/apache/spark/mllib/classification/LogisticRegression.scala

```diff
@@ -126,6 +137,35 @@ class LogisticRegressionModel (
       bestClass.toDouble
     }
   }
+
+  override def save(sc: SparkContext, path: String): Unit = {
```

Comment:
I'll save numFeatures and numClasses in all classification models' metadata.  I'm going for metadata instead of data in case the model data requires multiple RDD rows (e.g., for decision tree).